src/Judopay/Model/ResumeThreeDSecureTwo.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Judopay\Model;
+
+use Judopay\DataType;
+use Judopay\Model;
+
+class ResumeThreeDSecureTwo extends Model
+{
+    protected $resourcePath = 'transactions/receiptId/resume3ds';
+    protected $validApiMethods = array('update');
+
+    protected $attributes
+        = array(
+            'receiptId'            => DataType::TYPE_STRING,
+            'cv2'                  => DataType::TYPE_STRING,
+
+            // Inner objects
+            'threeDSecure'         => DataType::TYPE_THREE_D_SECURE_TWO,
```

Comment:
I think it would be better just to expose the methodCompletion string inside threeDSecure at this level (as was done in https://thedojo.atlassian.net/browse/JR-4653 for the DotNetSDK - but as that's a new request I'll raise it in a separate ticket)